db/migrate/20180108114505_create_restaurants.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.string :name
       t.string :cuisine
       t.decimal :rating, default: 0
-      t.boolean :accepts_10bis, default: false
+      t.boolean :accepts_ten_bis, default: false
```

Comment:
Once you committed and pushed a migration, you don't change it retroactively.

I won't ask you to re-do this here, but in real life you would create a new migration, and rename the column within.